racetrack_node:
  ros__parameters:
    start_pose:
      position: {x: 0, y: 0, z: 0}
      orientation: {x: 0, y: 0, z: 0, w: 1}
    
    gate_poses:
      - position: {x: 5, y: 5, z: 0}
        orientation: {x: 0, y: 0, z: 0, w: 1}
      - position: {x: 15, y: 15, z: 0}
        orientation: {x: 0, y: 0, z: 0, w: 1}
    
    num_laps: 3
